NSS-Day-Cohort-42/rare-API-media-moguls
# Rp post reactions

#### Changes Made
1. Added models for reactions and post reactions 
1. made made migrations and migrated new models
1. Added fixtures for reactions and post reactions and loaded data
1. made reaction.py file in views with list method and custom action for when a user reacts to a post. 
1. added some code to settings.py and urls.py to support images
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout rp-postReactions
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> Open the client side pull request rp-reactions. Go to the post detail page and make sure that a user can click on a reaction and that the count beside the emoji is incremented. The user should only be allowed to react to each emoji for a post once. 

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
